strawberry/lazy_type.py

```diff
@@ -15,6 +16,15 @@ class LazyType(Generic[TypeName, Module]):
     package: Optional[str]
```

Comment:
I saw that you had to add `None` to a lot of places defining `LazyType`. Maybe `package` could be set to `None` by default to avoid that?

Reply:
yes, good suggestion!